pkgs/development/libraries/rapidjson/git.nix

```diff
@@ -0,0 +1,50 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, fetchpatch
+, pkg-config
+, cmake
+, gtest
+, valgrind
+}:
+
+stdenv.mkDerivation rec {
+  pname = "rapidjson";
+  version = "unstable-2023-09-28";
+
+  src = fetchFromGitHub {
+    owner = "Tencent";
+    repo = "rapidjson";
+    rev = "f9d53419e912910fd8fa57d5705fa41425428c35";
+    hash = "sha256-rl7iy14jn1K2I5U2DrcZnoTQVEGEDKlxmdaOCF/3hfY=";
+  };
```

Comment:
I was confused by the owner here being `Tencent`, but on the other drv being `miloyip`. Could you update the other (i.e. `rapidjason/default.nix`) to use the Tencent repo as well?

They point to the same thing, since `miloyip` moved it to `Tencent` AFAICT.